pkg/validation/validate_cluster.go

```diff
@@ -199,13 +199,22 @@ func (v *ValidationCluster) collectComponentFailures(ctx context.Context, client
 	return nil
 }
 
+var masterStaticPods = []string{
+	"kube-apiserver",
+	"kube-controller-manager",
+	"kube-scheduler",
```

Comment:
Just curious, do `etcd-manager-*` startic pods get checked also?

Reply:
No. If the cluster uses the legacy etcd provider, there won't be any `etcd-manager-*` pods. Checking the cluster's spec would add a lot of complexity. I suspect the kube-apiserver pods won't be able to go into ready state without etcd pods.

Reply:
OK, probably not worth the trouble for now. Thanks.